### ISSUE TITLE
HTML being displayed onscreen in Reseller Product Widget Options

### ISSUE THREAD

**Issue by cberesford** [godaddy/wp-contact-widgets#43]
`selected='selected'` markup is being rendered in Appearance --> Widgets --> Reseller Product. See screenshot below

![screen shot 2017-06-20 at 11 24 36 pm](https://user-images.githubusercontent.com/8406886/27370131-02a1706a-5610-11e7-92dd-6d6eabd73f8b.png)

**Comment by EvanHerman** [comment]
Hi @cberesford ,

This looks to be more of an issue with our Reseller Plugin, and not necessarily a bug inside of the contact widgets plugin.

Let me do some quick testing and get back to you with some of my findings.

Thanks so much for taking the time to properly file a bug report here, it is greatly appreciated!


Evan

**Comment by cberesford** [comment]
Whoops, sorry about that - I totally got the repos mixed up for a second. Meant to file it into https://github.com/godaddy/wp-reseller-store :)

Thanks for taking care of this @EvanHerman

**Comment by EvanHerman** [comment]
No problem at all! I just copied this issue over to our reseller store https://github.com/godaddy/wp-reseller-store/issues/23

Along with that I went ahead and made some changes to the branch there (`embed-item`) which should address the issues outlined there (along with a few other enhancements). Feel free to pull down the updates from that branch and re-test. 

If you run into any other issues feel free to comment on the PR over on the reseller repo and I'll hop back on and get things cleared up.

Thanks again @cberesford !
